integration_tests/pages/appointmentTime.ts

```diff
@@ -23,7 +23,7 @@ export default class AppointmentTimePage extends Page {
 
   enterDate = (moment: Moment): AppointmentTimePage => {
     cy.get(this.inductionDateDayId).type(moment.date().toString())
-    cy.get(this.inductionDateMonthId).type(moment.month().toString())
+    cy.get(this.inductionDateMonthId).type((moment.month() + 1).toString())
```

Comment:
Broken because of the new year, January is entered as 0

Reply:
Ah, I also fixed in my branch too.